### DISCUSSION TITLE
研究生复盘 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#6] [General]
# 研究生复盘 | 高策

江湖小虾米

http://gaocegege.com/Blog/%E9%9A%8F%E7%AC%94/master

**Comment by Emanual20** [comment]
支持对“我个人建议可以考虑读一个研究生”两类人的观点，本人感觉是第一类，现在研一升研二的暑假深切地感受到对科研的无力…总感觉还不如本科直接去找个工作来的划算:( anyway人生确实要向前！

**Reply by gaocegege** [reply]
保持好心态

**Comment by mtdww** [comment]
又看到了这篇文章，其实看到这篇文章是去年的事。当时还在上班，是华为od的测试。一直觉得没有意思，看不到希望，然后就想着考个研究生再试试。现在已经顺利读上研究生了，但是好像感觉还不如工作，可能是读研太累了，就感觉人很矛盾。

**Reply by gaocegege** [reply]
再坚持看看吧

**Reply by mtdww** [reply]
好的。在坚持呢。💪
